### PR TITLE
sdk 5.2.0 support

### PR DESCRIPTION
**Pull request by gluhar2006**

### CODE REVIEW

File: tests/test_estimate_descriptor.py [hunk 1]
```diff
@@ -26,9 +26,9 @@
 from tests.detect_test_class import VLIMAGE_SMALL
 from tests.resources import WARP_WHITE_MAN, HUMAN_WARP, WARP_CLEAN_FACE, BAD_THRESHOLD_WARP
 
-EFDVa = EXISTENT_FACE_DESCRIPTOR_VERSION_ABUNDANCE = [54, 56, 57, 58]
+EFDVa = EXISTENT_FACE_DESCRIPTOR_VERSION_ABUNDANCE = [54, 56, 57, 58, 59]
 
-EHDVa = EXISTENT_HUMAN_DESCRIPTOR_VERSION_ABUNDANCE = [DHDV]
+EHDVa = EXISTENT_HUMAN_DESCRIPTOR_VERSION_ABUNDANCE = [DHDV, 102, 103]
```
Comment (matemax) [inline tests/test_estimate_descriptor.py]: ...

File: lunavl/sdk/detectors/base.py [hunk 2]
```diff
@@ -157,9 +157,11 @@ def validateBatchDetectInput(
         validationError, imageErrors = detector.validate(coreImages, detectAreas, limit)
     if validationError.isOk:
         return
-    # uncomment after FSDK-2930
-    # if validationError.error != FSDKError.ValidationFailed:
-    #     raise LunaSDKException(LunaVLError.fromSDKError(validationError), errors)
+    if validationError.error != FSDKError.ValidationFailed:
+        raise LunaSDKException(
+            LunaVLError.ValidationFailed.format("Failed validation"),
```
Comment (matemax) [inline lunavl/sdk/detectors/base.py]: if error is not ValidationFailed raise ValidationFailed? I think that is strange.

File: lunavl/sdk/detectors/facedetector.py [hunk 3]
```diff
@@ -307,17 +308,19 @@ def _validateReDetectInput(self, coreImages: List[CoreImage], detectAreas: List[
...
         else:
             validationError, imagesErrors = self._detector.validate([coreImages], [[detectAreas]])
         if validationError.isOk:
             return
-        # wait FSDK-2930
-        # if validationError.error != FSDKError.ValidationFailed:
-        #     raise LunaSDKException(LunaVLError.fromSDKError(validationError), imagesErrors)
+        if validationError.error != FSDKError.ValidationFailed:
+            raise LunaSDKException(
+                LunaVLError.ValidationFailed.format("Failed validation"),
```
Comment (matemax) [inline lunavl/sdk/detectors/facedetector.py]: strange logic

File: tests/test_estimate_descriptor.py (hunk 1 above)
Comment (matemax) [reply tests/test_estimate_descriptor.py]: If DHDV =102 what happens?

**Review by matemax — APPROVED** [review]

File: tests/test_estimate_descriptor.py (hunk 1 above)
Comment (gluhar2006) [reply tests/test_estimate_descriptor.py]: updated

File: lunavl/sdk/detectors/base.py (hunk 2 above)
Comment (gluhar2006) [reply lunavl/sdk/detectors/base.py]: updated

File: lunavl/sdk/detectors/facedetector.py (hunk 3 above)
Comment (gluhar2006) [reply lunavl/sdk/detectors/facedetector.py]: updated

**Comment by matemax** [comment]
up version please

**Comment by gluhar2006** [comment]
> up version please

done

**Pull request merged**